src/renderer/components/dex/OrderForm.vue

```diff
@@ -396,7 +404,31 @@ export default {
       return (totalMultiple / quantity).toString();
     },
 
+    validateQuantity() {
+      if (!this.$store.state.orderQuantity
+        || this.$store.state.orderQuantity === ''
+        || this.$store.state.orderQuantity[this.$store.state.orderQuantity - 1] === '.') {
+        return;
+      }
+
+      const marketTickSizeDecimals = this.currentMarket.minimumTickSize.toString().split('.')[1].length;
+      const allowedQuantityDecimals = 8 - marketTickSizeDecimals;
+      const decimalFactor = 10 ** allowedQuantityDecimals;
```

Comment:
Can you put these two ints into variables with meaningful names, or the constants file?